### DISCUSSION TITLE
High level visualisation

### DISCUSSION THREAD

**Discussion by Hoogkamer** [edmcouncil/fibo#1828] [General]
Hi, I was missing a total overview of the fibo ontologies. Maybe it is there, but I could not find it. 
Anyway, I created one. Maybe handy for others also.

![fibo overview](https://user-images.githubusercontent.com/17671871/190597657-7b6e4474-2247-4ed7-b86a-97238ebb993f.png)

**Comment by mereolog** [comment]
@Hoogkamer thanks for that.
What visualisation tool did you use?

**Comment by Hoogkamer** [comment]
@mereolog I use the cytoscapejs library for that, and built an application around that to visualise/edit graphs: https://termboard.com/app#/?demo=fibo
